src/containers/Router/Routes/RestrictedRoute/RestrictedRoute.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from "react";
+import { Navigate, useParams } from "react-router-dom";
+
+import { getCookies } from "@/src/utils";
+import { links } from "@/src/types";
+
+import { IRestrictedRouteProps } from "../types/Routes.type";
+
+const RestrictedRoute: FC<IRestrictedRouteProps> = ({
```

Comment:
That seems a good solution for restricting the users from sensitive actions. But I wonder what you think of reading sensitive cookies on the client. I think it is still a good solution because no one will ever try to steal sensitive data from our users, but I want to know if you think it might be an issue in future (since I have to write the risk log XD)